tools/markdown-magic/src/md-magic.config.cjs

```diff
@@ -106,19 +106,15 @@ const generateTrademarkSection = (headingOptions) =>
 	});
```

Comment:
Plans for future improvements (in subsequent PRs):
1. By default, don't include the following sections for `@fluid-private` and `@fluid-internal` packages:
    - Installation instructions
    - API docs links
2. By default, configure the installation instructions section's "devDependency" argument based on whether or not the package is in the `@fluid-tools` scope (can still be overridden)